src/app/organisations/postEditOrganisation.js

```diff
@@ -0,0 +1,40 @@
+const { sendResult } = require("../../infrastructure/utils");
+const {
+  getOrganisationByIdV2,
+} = require("./../../infrastructure/organisations");
+
+const postEditOrganisation = async (req, res) => {
+  const organisation = await getOrganisationByIdV2(req.params.id, req.id);
+  const { name, address } = req.body;
+
+  const view = "organisations/views/editOrganisation";
+  const regex = /[±!@£$%^*_+§¡€#¢§¶•ªº«\\/<>:;|=.~"]/;
+  const model = {
+    csrfToken: req.csrfToken(),
+    organisation,
+    validationMessages: {},
+  };
+
+  if (!name.trim() && !address.trim()) {
```

Comment:
Can you refactor the validation into a separate function?  Something like this https://github.com/DFE-Digital/login.dfe.support/blob/main/src/app/organisations/postCreateOrganisation.js#L69

It's a personal opinion (so tell me if I'm being unresaonable) but I think that if the validation is more than 1 simple thing, then putting it in its own function makes it easier to read later on